cernopendata/templates/cernopendata_records_ui/records/record_detail.html

```diff
@@ -103,6 +103,24 @@ <h2 id="previewer">Preview</h2>
         {%include "cernopendata_ispy/ispy_standalone.html"%}
     {% endif %}
 
+    {% if record.cross_section %}
+    <h2>Cross section</h2>
+        {% if record.cross_section.description %}
+            <span>{{record.cross_section.description | safe}}</span>
+            <br>
+        {% else %}
+            <p>For {{record.collision_information.type | safe}} collisions at {{record.collision_information.energy | safe}}, this sample has a cross section of (<span><strong>{{record.cross_section.total_value | safe}}</strong></span> +/- <span><strong>{{record.cross_section.total_value_uncertainty | safe}}</strong></span>) pb, calculated using the method described in <a href="/docs/cms-guide-cross-sections">CMS guide for cross sections</a>.</p>
```

Comment:
Some comments for the template:

- The `| safe` part should not be necessary in many places here, since we won't have HTML in the values. We should probably keep it for the free-text description subfield, but for other subfields it may be probably removed. 
- If some of the subfields may not be always present (say filter efficiency?), we could change the template to the style of `record.get('cross_section', {}).get(''filter_efficiency', "") ` to make sure we have some default value even if a subfield is not present. Would this be useful?

BTW we don't have cross_sections defined in the metadata model. I guess it may be useful? If so, we could add it during rebase.

Reply:
We can link to
`<a href="/docs/cms-guide-cross-sections#run2">CMS guide for cross sections</a>`

Reply:
Added.